apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/tab/TabController.java

```diff
@@ -36,4 +35,16 @@ public CreateTabResponse createTab(
         return creator.createTab(page, userId, request);
     }
 
+    @DeleteMapping("deletetab")
+    @ResponseBody
+    public DeleteTabResponse deleteTab(@PathVariable("page") Long page, @RequestBody DeleteTabRequest request) {
+        return creator.deleteTab(page, request);
+    }
+
+    @PutMapping("updatetab")
+    @ResponseBody
+    public UpdateTabResponse updateTab(@RequestBody UpdateTabRequest request) {
```

Comment:
standard convention: 

```java
@PutMapping("{tab}")
public UpdateTabResponse updateTab(@PathVariable("tab") Long tab, @RequestBody UpdateTabRequest request)
```

and remove the tabId from the `UpdateTabRequest` object